atuin-client/src/settings.rs

```diff
@@ -19,9 +19,19 @@ pub enum SearchMode {
     FullText,
 }
 
+// FIXME: Can use upstream Dialect enum if https://github.com/stevedonovan/chrono-english/pull/16 is merged
+#[derive(Clone, Debug, Deserialize, Copy)]
+pub enum Dialect{
+    #[serde(rename = "us")]
+    Us,
+
+    #[serde(rename = "uk")]
+    Uk,
+}
```

Comment:
I think it would be better to implement the Into<Dialect> for this type https://doc.rust-lang.org/std/convert/trait.Into.html